company/remote/index.md

```diff
@@ -37,6 +37,16 @@ At Sourcegraph, we support these kinds of interactions in many ways. Here are a
 
 The fundamentals of supporting teammates (regardless of their experience level) are the same on all-remote teams, it is just that the processes are different. Video chats can be arranged just as easily as stopping by someone's desk to introduce yourself or showing up at someone's office to express informal concerns.
 
+### Is there an expectation to work over the weekends?
+
+> Why there is so much activity over the weekend (Github, PRs, Slack)? I felt a little bit of pressure not working.
+
+Some people prefer to work in a really flexible way, which can sometimes mean that they want to hack on something on a Saturday instead of a Friday afternoon, for example. Others prefer a set schedule. Both are valid options
```

Comment:
```suggestion
Some people prefer to work in a really flexible way, which can sometimes mean that they want to hack on something on a Saturday instead of a Friday afternoon, for example. Others prefer a set schedule. Both are valid options.
```